chipsec/helper/osx/helper.py

```diff
@@ -48,6 +51,23 @@ def __init__(self):
         self.os_uname   = platform.uname()
         self.dev_fh = None
 
+    def load_driver(self):
+        driver_path = os.path.join(chipsec.file.get_main_dir(), "chipsec",
+                                   "helper", "osx", self.DRIVER_NAME)
+        # Make sure the driver image and its subdirectories are owned by root.
+        s = os.stat(driver_path)
+        if s.st_uid != 0 or s.st_gid != 0:
+            os.chown(driver_path, 0, 0)
+            for root, dirs, files in os.walk(driver_path):
+                for f in dirs + files:
+                    os.chown(os.path.join(root, f), 0, 0)
+        subprocess.check_call(["kextload", driver_path])
+        if os.path.exists(self.DEVICE_NAME):
```

Comment:
I think you should look at the return code of the subprocess call to make a decision if the module is loaded or not.